api/imagebuilder/janitor_test.go

```diff
@@ -0,0 +1,27 @@
+package imagebuilder
+
+import (
+	"testing"
+	"time"
+
+	"github.com/gojek/merlin/cluster/mocks"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
+)
+
+func TestJanitor_CleanJobs(t *testing.T) {
```

Comment:
Can you add test that prove it will only delete Merlin's image build jobs older than the retention period?